cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/AppLifecycleHttpHandler.java

```diff
@@ -254,7 +224,7 @@ public BodyConsumer createAppVersion(HttpRequest request, HttpResponder responde
     ApplicationId applicationId = validateApplicationVersionId(namespaceId, appId, versionId);
 
     try {
-      return deployAppFromArtifact(applicationId);
+      return deployApplicationFromArtifact(applicationId);
```

Comment:
Why we have renamed this method ? This introduces unneeded diff in the PR

Reply:
renamed it back to `deployAppFromArtifact`